1.9.0

```diff
@@ -1,5 +1,21 @@
 {
   "instructionList":[
+    {
+       "instructionTypeName": "recordFieldMapper",
+       "taskList": [
+         {
+           "jiraTicket": "ac-57039",
+           "description": "Map data from FinServ Description to custom Description field",
+           "actionType": "RecordFieldMapper",
+           "actionTypeName": "Query",
+           "optionsMap": {
+             "incomingObject": "FinServ__FinancialAccount__c",
+             "sourceField": "FinServ__Description__c"
```

Comment:
Need a comma at the end here. Make sure you are fully testing your instructions before submitting a PR.

Reply:
Comma added